src/environments/environment.prod.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
-  production: true,
-  backendApi: 'http://174.138.57.106:81',
-  submissionApi: 'http://174.138.57.106:82',
-};
+  production: false,
```

Comment:
Not sure if you meant this or not, but I think we should probably keep 'production' at 'true' here